docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Feature HowTos
+description: This section lists documents inside the Feature HowTos section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
```

Comment:
Consider moving these two HowTos to respective PBCs

Reply:
Out of scope of this task